packages/blocks/src/api/templates.js

```diff
@@ -104,7 +105,7 @@ export function synchronizeBlocksWithTemplate( blocks = [], template ) {
 			};
 
 			const normalizedAttributes = normalizeAttributes(
-				get( blockType, [ 'attributes' ], {} ),
+				blockType?.attributes ?? {},
```

Comment:
Adding a note that this will behave differently if `attributes` is `null` here, but as far as I can tell, we don't expect that to be a valid value 👍

Reply:
Actually, we might see `null`  coming from the server. The question is if it’s mapped correctly before registering the block.

Reply:
Related code in WP core:

https://github.com/WordPress/wordpress-develop/blob/b351a2f05805cf2d8b9ee431020abbe692b8fb36/src/wp-includes/class-wp-block-type.php#L149

I see we provide an array later in:

https://github.com/WordPress/wordpress-develop/blob/b351a2f05805cf2d8b9ee431020abbe692b8fb36/src/wp-includes/class-wp-block-type.php#L370

We also ignore `null` values passed from the server:

https://github.com/WordPress/gutenberg/blob/2c8ac507c1800b1e3a0fa272d65bc2a03a34afb6/packages/blocks/src/api/registration.js#L176L183

We good 😃